scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -249,10 +256,20 @@ build-provider-create $bartender_name
   then
     cp -aR "$HOOK_EXTRAS_DIR" $temp_dir/extras
   fi
+
+  if [ -n "$LIVECD_ROOTFS_DIR" ]
+  then
+    cp -aR "$LIVECD_ROOTFS_DIR" $temp_dir/livecd-rootfs
```

Comment:
This isn't a blocker but it might be worth putting braces around `$temp_dir` here.